src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderNotification.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.divorce.common.notification;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.notification.ApplicantNotification;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+import uk.gov.hmcts.divorce.notification.NotificationService;
+
+import java.util.Map;
+
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.divorce.notification.CommonContent.IS_REMINDER;
+import static uk.gov.hmcts.divorce.notification.CommonContent.SOLICITOR_NAME;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.CITIZEN_APPLY_FOR_CONDITIONAL_ORDER;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.SOLICITOR_AWAITING_CONDITIONAL_ORDER;
+
+@Component
+@Slf4j
+public class AwaitingConditionalOrderNotification implements ApplicantNotification {
+
+    @Autowired
+    private CommonContent commonContent;
+
+    @Autowired
+    private NotificationService notificationService;
+
+    @Override
+    public void sendToApplicant1(final CaseData caseData, final Long id) {
+        log.info("Notifying applicant 1 that they can apply for a conditional order: {}", id);
+        final Applicant applicant1 = caseData.getApplicant1();
+        final Map<String, String> templateVars = commonContent
+            .conditionalOrderTemplateVars(caseData, id, applicant1, caseData.getApplicant2());
+        templateVars.put(IS_REMINDER, CommonContent.NO);
+
+        notificationService.sendEmail(
+            applicant1.getEmail(),
+            CITIZEN_APPLY_FOR_CONDITIONAL_ORDER,
+            templateVars,
+            applicant1.getLanguagePreference()
+        );
+    }
+
+    @Override
+    public void sendToApplicant1Solicitor(final CaseData caseData, final Long id) {
+        log.info("Notifying applicant 1 solicitor that they can apply for a conditional order: {}", id);
+
+        final Applicant applicant1 = caseData.getApplicant1();
+        final Map<String, String> templateVars = commonContent.basicTemplateVars(caseData, id);
+        templateVars.put(SOLICITOR_NAME, applicant1.getSolicitor().getName());
+
+        notificationService.sendEmail(
+            applicant1.getSolicitor().getEmail(),
+            SOLICITOR_AWAITING_CONDITIONAL_ORDER,
+            templateVars,
+            applicant1.getLanguagePreference()
+        );
+        log.info("Successfully sent awaiting conditional order notification for case : {}", id);
```

Comment:
this last log statement is present in some methods but not in all so would be nice to make it consistent

Reply:
looks like one I forgot to remove, I will change